### DISCUSSION TITLE
Supported Postgres versions - clarifications needed

### DISCUSSION THREAD

**Discussion by labou77** [Unleash/unleash#8261] [Q&A] [answered]
Dear Unleash community,
I would like to get a clear picture about Postgres versions supported vs Unleash version. I find some contradictions across the documentation :

- This page mentions Postgres version 10 or later, no Unleash release specified (https://docs.getunleash.io/using-unleash/deploy/getting-started)
- This page mentions Postgres version 14.x or newer, no Unleash release specified (https://docs.getunleash.io/using-unleash/deploy/database-setup)
- This page mentions Postgres version 13 or higher from Unleash 6.0.0 (https://github.com/Unleash/unleash/releases/tag/v6.0.0)

How could I get a clear picture about the Postgres version supported so I can define my Unleash upgrade path?

Thanks a lot.

**Comment by chriswk** [comment, marked as answer]
Hi @labou77 - We'll update our docs, to make one unified place to check which minimum version you should / can depend on.

If you define your version to be at least 13, you should be good until its End of Life in November 2025. 
However, if you have the option, I'd look at using last year's 16, since that has EOL in 2028.

Unleash currently hosts our SAAS on v14 instances, but we test migrations on 13 and up.

I've made https://github.com/Unleash/unleash/pull/8276 in order to unify/clarify that the recommended version is 13 or newer.
